CHANGELOG.md

```diff
@@ -10,15 +10,36 @@ nav_order: 1
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
 
 - Add support for `autoscaler` service integration
-- Add `aiven_opensearch` resource field `opensearch_user_config.azure_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
-- Add `aiven_opensearch` resource field `opensearch_user_config.gcs_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
-- Add `aiven_opensearch` resource field `opensearch_user_config.s3_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
-- Add `aiven_opensearch` datasource field `opensearch_user_config.gcs_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
-- Add `aiven_opensearch` datasource field `opensearch_user_config.azure_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
-- Add `aiven_opensearch` datasource field `opensearch_user_config.s3_migration.include_aliases`: Whether to restore aliases alongside their associated indexes
+- Add `aiven_opensearch` resource field `opensearch_user_config.azure_migration.include_aliases`: Whether to restore
+  aliases alongside their associated indexes
+- Add `aiven_opensearch` resource field `opensearch_user_config.gcs_migration.include_aliases`: Whether to restore aliases
+  alongside their associated indexes
+- Add `aiven_opensearch` resource field `opensearch_user_config.s3_migration.include_aliases`: Whether to restore aliases
+  alongside their associated indexes
+- Add `aiven_opensearch` datasource field `opensearch_user_config.gcs_migration.include_aliases`: Whether to restore
+  aliases alongside their associated indexes
+- Add `aiven_opensearch` datasource field `opensearch_user_config.azure_migration.include_aliases`: Whether to restore
+  aliases alongside their associated indexes
+- Add `aiven_opensearch` datasource field `opensearch_user_config.s3_migration.include_aliases`: Whether to restore
+  aliases alongside their associated indexes
 - Change `aiven_cassandra` resource field `cassandra_user_config.additional_backup_regions`: remove deprecation
 - Change `aiven_cassandra` datasource field `cassandra_user_config.additional_backup_regions`: remove deprecation
 
+## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
```

Comment:
There are now two `[MAJOR.MINOR.PATCH]` headings, is that on purpose? Another one on L10.